quickstart/pg-init/multi-psql-db.sh

```diff
@@ -9,10 +9,10 @@ set -u
 function create_user_and_database() {
 	local database=$1
 	local username=$2
-	local password=$3
+	local password="'$3'"
 	echo "  Creating user '$username' and database '$database'"
 	psql -v ON_ERROR_STOP=1 --username "$POSTGRES_USER" <<-EOSQL
-	    CREATE USER "$username" WITH PASSWORD '$password';
+	    CREATE USER "$username" WITH PASSWORD $password;
```

Comment:
`$password` should be in double quotes like `$username` to prevent shell interpretation of spaces, etc.

Reply:
@synackd I did try double quoting `$password` but psql didn't like that for some reason, so figured for now removing the quotes would at least allow the variable to be resolved.

Reply:
That's strange. Does it have something to do with the particular password you're using? The quotes should go away after the shell interprets them in the script so that the value itself is passed to `psql`.